Manager/PostManager.php

```diff
@@ -114,4 +114,19 @@ public function getThumbnailPath(Post $post)
 
         return $post->getGuid();
     }
+    
+    /**
+     * @param \DateTime $date
+     * @return array
+     */
+    public function findByDate(\DateTime $date)
+    {
+        $qb = $this->repository->createQueryBuilder('c');
```

Comment:
Yes, it should be better and I agree for the date sort too. I will update my comment.